organizations/tests.py

```diff
@@ -1 +1,17 @@
-# Create your tests here.
+from plio.tests import BaseTestCase
+from organizations.models import Organization
+from rest_framework import status
+from django.urls import reverse
+
+
+class OrganizationCRUDTestCase(BaseTestCase):
+    def setUp(self):
+        super().setUp()
+        # seed some organizations
+        Organization.objects.create(name="Plio 1", shortcode="org-1")
+        Organization.objects.create(name="Plio 2", shortcode="org-2")
+
+    def test_a_normal_user_cannot_list_organizations(self):
+        # get organizations
```

Comment:
We can mention the desired output in the docstring if it becomes too verbose
```suggestion
    def test_normal_user_list_organizations(self):
        """A non-superuser should not be able to list organizations""'
        # get organizations
```